src/modules/theme/constants/colors.js

```diff
@@ -8,7 +8,8 @@ export type ColorsType = {
   textSecondaryColor: string,
   textDecorationColor: string,
   textDisabledColor: string,
-  backgroundColor: string
+  backgroundColor: string,
+  tuNewsColor: string
```

Comment:
I feel that tuNewsColor should not exist here. 

This file is more for general 'theme' colors for the whole app, whereas Tu News is a very specific element that exists in one or two places only.

What does everybody think?